src/EnergyPlus/HVACManager.cc

```diff
@@ -2051,7 +2051,7 @@ void ResolveLockoutFlags(EnergyPlusData &state, bool &SimAir) // TRUE means air
     }
 }
 
-void ResetHVACControl(EnergyPlusData &state)
+void ResetHVACControl(EnergyPlusData const &state)
```

Comment:
[src/EnergyPlus/HVACManager.cc:2054]:(style),[constParameter],Parameter 'state' can be declared as reference to const